cypress/integration/chi-vue/data-table.spec.js

```diff
@@ -1175,6 +1190,238 @@ describe('Data Table', () => {
+                .contains(`${paginationResults} results`);
+              cy.get(`[data-cy='data-table-bulk-actions']`)
+                .find(`.${BULK_ACTIONS_CLASSES.BULK_ACTIONS} .${BULK_ACTIONS_CLASSES.START} 
+                  .${BULK_ACTIONS_CLASSES.LABEL}`)
+                .contains(`Actions (${selected} Selected)`);
+              cy.get("@rows").should("have.length", 1);
+              expect(spy).to.be.calledOnce;
+            });
+        })
+    });
+
+    it('Should deselect "Show selected only" and show data table is still in its original state', () => {
+      cy.window()
+        .its('dataTableBulkActionsExample')
+        .then(dataTableBulkActionsExample => {
+          const paginationResults = '6';
+          const selected = '1';
+          const spy = cy.spy();
+          const dataTableRef = dataTableBulkActionsExample.$refs.dataTableBulkActionsRef;
+
+          dataTableRef.$on(`${DATA_TABLE_EVENTS.BULK_ACTIONS.SHOW_SELECTED_ONLY}`, spy);
+          cy.get(`[data-cy='data-table-bulk-actions']`)
+            .find(`.${BULK_ACTIONS_CLASSES.BULK_ACTIONS} .${BULK_ACTIONS_CLASSES.START} 
+            .${CHECKBOXES_CLASSES.LABEL}`)
+            .click()
+            .then(() => {
+              cy.get(`[data-cy='data-table-bulk-actions']`)
+                .find(`.${DATA_TABLE_CLASSES.SELECTABLE}`)
+                .as('selectablesAfterClick');
+              cy.get('@selectablesAfterClick')
+                .eq(1)
+                .find('input')
+                .should('be.checked');
+              cy.get('@selectablesAfterClick')
+                .eq(2)
+                .find('input')
+                .should('not.be.checked');
+              cy.get(
+                `[data-cy='data-table-bulk-actions'] .${PAGINATION_CLASSES.PAGINATION} 
+                    .${PAGINATION_CLASSES.RESULTS}`
+              )
+                .find('span')
+                .contains(`${paginationResults} results`);
+              cy.get(`[data-cy='data-table-bulk-actions']`)
+                .find(`.${BULK_ACTIONS_CLASSES.BULK_ACTIONS} .${BULK_ACTIONS_CLASSES.START} 
+                  .${BULK_ACTIONS_CLASSES.LABEL}`)
+                .contains(`Actions (${selected} Selected)`);
+              cy.get("@rows").should("have.length", 3);
+              expect(spy).to.be.calledOnce;
+            });
+        });
+    });
+
+    it('Should close bulk actions and show data table is still in its original state', () => {
+      cy.window()
+        .its('dataTableBulkActionsExample')
+        .then(dataTableBulkActionsExample => {
+          const spy = cy.spy();
+          const dataTableRef = dataTableBulkActionsExample.$refs.dataTableBulkActionsRef;
+          const paginationResults = '6';
+
+          dataTableRef.$on(`${DATA_TABLE_EVENTS.BULK_ACTIONS.CANCEL}`, spy);
+          cy.get(`[data-cy='data-table-bulk-actions']`)
+            .find(`.${BULK_ACTIONS_CLASSES.BULK_ACTIONS} .${BULK_ACTIONS_CLASSES.END}`)
+            .click()
```

Comment:
Is this clicking the actual X button? Seems like the click happens on `.${BULK_ACTIONS_CLASSES.END}`.